spirv_glsl.cpp

```diff
@@ -1059,6 +1059,9 @@ void CompilerGLSL::replace_fragment_output(SPIRVariable &var)
 		if (location != 0)
 			throw CompilerError("Arrayed output variable used, but location is not 0. "
 			                    "This is unimplemented in SPIRV-Cross.");
+		
+		if (is_legacy_es())
```

Comment:
In that case, wouldn't this be needed in non-arrayed output as well, where location != 0?